test/job/os_versions_test.exs

```diff
@@ -0,0 +1,49 @@
+defmodule Bob.Job.OSVersionsTest do
+  use ExUnit.Case, async: true
+
+  alias Bob.Job.OSVersions
+
+  import Mock
```

Comment:
I would like to avoid mocking if possible. Instead you can add a `OSVersions.select_os_versions(tags)` function that takes the result of `Bob.DockerHub.fetch_repo_tags/1` and test that.

Reply:
sounds good, I didn't get around to changing this today, but I'll look to do this another day.